src/main/java/org/cyclops/cyclopscore/client/gui/config/GuiConfigOverviewBase.java

```diff
@@ -28,7 +29,7 @@
 	 */
 	public GuiConfigOverviewBase(ModBase mod, GuiScreen parentScreen) {
 		super(parentScreen, getConfigElements(mod), mod.getModId(), false, false,
-			GuiConfig.getAbridgedConfigPath(mod.getConfigHandler().getConfig().toString()));
+		        mod.getModName() + " " + L10NHelpers.localize("config.cyclopscore"));
```

Comment:
I would add the mod name as a parameter to the localization, because in some languages the mod name could be placed _after_ "configuration". (using %s in the l10n value).